converters/fromZigbee.js

```diff
@@ -4052,7 +4370,30 @@ const converters = {
             const action = actionLookup[value];
 
             if (button) {
-                return {click: button + (action ? `_${action}` : '')};
+                return {action: button + (action ? `_${action}` : '')};
```

Comment:
Change to `{action: getProperty(action, msg, model)}`. Make sure to also add `meta: {multiEndpoint: true}` to `ptvo.switch` and `DIYRuZ_R8_8` in `devices.js`